gradle.properties

```diff
@@ -189,7 +189,7 @@ jerseyVersion=2.27
 jradiusVersion=jradius-1.1.5
 
 axetWgetVersion=1.4.9
-httpclientVersion=4.5.5
+httpclientVersion=4.5.6
```

Comment:
Is there a particular reason for this one?

Reply:
Some fixes : https://archive.apache.org/dist/httpcomponents/httpclient/RELEASE_NOTES-4.5.x.txt